InnoClinic.AuthorizationAPI/API/API.csproj

```diff
@@ -7,8 +7,9 @@
   </PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.0" />
-		<PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.0">
+		<PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="8.0.14" />
```

Comment:
Are you sure that we need this package here? As you already have it in DAL and you added DAL into dependencies, this package will be also applied for API